backend/app/app.js

```diff
@@ -1,11 +1,41 @@
-var express = require('express');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
-var path = require('path');
-var logger = require('morgan'); // not yet used but may be needed in the future
-var config = require('./config'); // this will be the place where configuration is stored
+var express = require("express");
+var cookieParser = require("cookie-parser");
+var bodyParser = require("body-parser");
+var path = require("path");
+var logger = require("morgan"); // not yet used but may be needed in the future
+//var config = require("./config"); // this will be the place where configuration is stored
+var passport = require("passport");
+var Strategy = require("passport-local").Strategy;
+var mongoose = require("mongoose");
 
-// var mongoose = require('mongoose'); not good enough docu. can we use mongojs or the native mongodb driver instead?
+// DB initialization
+var User = require("./models/User.model");
+var db = "mongodb://localhost/sprintr";
+
+mongoose.connect(db); // check for authentication here

```

Comment:
for the MVP, I think yes. technically it was stored on Users collection on sprintr Database on mongodb. lookups will be done on the Users collection. Basic schema is just username, password, team. Should the need to add more arises, we can just add it on the model.